lib/shortcodes/property-attribute.php

```diff
@@ -70,7 +70,13 @@ public function __construct() {
                 'name' => __( 'Strip Tags', ud_get_wp_property()->domain ),
                 'description' => __( 'Strip tags', ud_get_wp_property()->domain ),
                 'default' => ''
-              )
+              ),
+              'show_taxonomy_label'=> array(
+                'name' => __( 'Show Taxanomy Label', ud_get_wp_property()->domain ),
+                'description' => __( 'True/False : Show/Hide taxanomy label', ud_get_wp_property()->domain ),
+                'type' => 'text',
+                'default' => true
```

Comment:
Default value set to false for "show_taxanomy_label" in "property_attribute" shortcode.
Please note that it still defaults to TRUE in "property_meta" to support the previous behaviour.